lib/js/src/compose/types/page-block/base.ts

```diff
@@ -101,6 +101,12 @@ export class PageBlock {
   setTempID (): void {
     this.meta.tempID = this.meta.tempID || generateUID()
   }
+
+  reset (): void {
```

Comment:
Implement the clone function instead of this that will return a cloned class of this instance with a default blockID, tempID